src/control/Control.cpp

```diff
@@ -2468,6 +2466,19 @@ void Control::applyPreferredLanguage() {
 #endif
 }
 
+void Control::initButtonTool() {
+    vector<Buttons> buttons{Buttons::BUTTON_ERASER, Buttons::BUTTON_STYLUS, Buttons::BUTTON_STYLUS2,
+                            Buttons::BUTTON_MIDDLE, Buttons::BUTTON_RIGHT,  Buttons::BUTTON_TOUCH};
```

Comment:
I used the names that were already in SettingsEnum to not add to much changes in this already quite big PR. But I can do that

Reply:
I'll update `BUTTON_STYLUS` and `BUTTON_STYLUS2` to `BUTTON_STYLUS_ONE` and `BUTTON_STYLUS_TWO` respectively as this was also something I didn't like to much about the previous naming scheme.

Reply:
Should I also rename `BUTTON_ERASER` to `BUTTON_STYLUS_ERASER` while I'm at it, or does that not make sense?

Reply:
Better touch everything that is connected in one PR than have two PRs for one change. I'd leave the eraser as it is as it may be theoretically a different device (even though it is often integrated with the stylus).